### PR TITLE
Update VD POC

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
# Objective
This PR aims to enhance the precision and usability of the Proof of Concept (POC) documentation for our Vulnerability Detection system. By expanding the documentation to include more detailed sections and illustrative examples, we intend to provide clearer guidance and better support for users engaging with our vulnerability detection tools.

## Description
- Feedback on our existing POC documentation for the Vulnerability Detection system indicated that QA/users often encounter ambiguities and lack sufficient detail to effectively utilize the system. To address these concerns, we have undertaken a comprehensive enhancement of the documentation.
- Key improvements include:
  - **Expanded Sections**: Each section of the documentation now contains a more detailed description, outlining the purpose and functionality with greater clarity. This includes elaborating on the system's architecture, configuration options, and operational procedures.
  - **Illustrative Examples**: We've added practical examples to demonstrate common use cases and scenarios. These examples are designed to help users better understand how to configure and use the system for optimal vulnerability detection.
  - **Enhanced Formatting**: The documentation is formatted in Markdown for improved readability and easier navigation. This includes the use of headers, lists, code blocks, and tables where appropriate, making the content more accessible and user-friendly.

### CODE REVIEW

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 1]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
+
+Test the configuration
+----------------------
+
+You don’t have to perform any action. It performs vulnerability detection scans periodically for applications and operating systems on each monitored endpoint. In this example we use the installed packages on the Ubuntu endpoint.
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:57]:
```suggestion
You don’t have to perform any action. Wazuh detects the vulnerable packages installed in the Ubuntu endpoint automatically. The time it takes to detect vulnerabilities depends on the Syscollector update interval configured in the ``ossec.conf`` file.
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 2]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
+
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:58]:
```suggestion
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 3]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
+.. note::
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:59]:
```suggestion
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 4]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
+   The time it takes to detect vulnerabilities depends on the syscollector update interval configured in the ``ossec.conf`` file.
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:60]:
```suggestion
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 1 above)
Comment (javimed) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst:57]: Consider adding a link to the interval syscolletor option referred.

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 5]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
+
 Visualize the alerts
 --------------------
 
-You can visualize the alert data in the Wazuh dashboard. To do this, go to the Vulnerability Detection module, select Events, and click on any vulnerability to expand the document.
+You can visualize the alert data in the Wazuh dashboard. To do this, go to the Vulnerability Detection module, select Inventory, and see all vulnerable packages. Use the search bar to filter the results.
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:65]:
```suggestion
You can visualize detected vulnerabilities data in the Wazuh dashboard. Go to **Vulnerability Detection** and select **Inventory** to see a list of all vulnerable packages detected. Add ``firefox`` in the search bar to filter vulnerable Firefox packages.
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 6]
```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
 
-.. thumbnail:: /images/poc/vulnerabilities-ubuntu-alerts.png
-   :title: Detected vulnerabilities on Ubuntu alerts
+.. thumbnail:: /images/poc/vulnerabilities-inventory.png
+   :title: Detected vulnerabilities on Ubuntu, firefox example
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:68]:
```suggestion
   :title: Detected vulnerabilities on Ubuntu. Vulnerable Firefox packages example
```

**Review by javimed — CHANGES_REQUESTED** [review]
The related #7183 issue requires steps to install vulnerable packages in order to trigger alerts. We're not installing a vulnerable Firefox version but using one installed by default. We're not patching or uninstalling Firefox to show solved vulnerabilities either. Are we ok with this approach? Aren't we going to explain in the "Test the configuration" sub-section that installing Firefox 116 or similar is required to see the alerts?

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 2 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst:64]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 5 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 6 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 4 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 3 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst (hunk 1 above)
Comment (GabrielEValenzuela) [reply source/proof-of-concept-guide/poc-vulnerability-detection.rst]: Done!

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 7]
```diff
@@ -51,12 +52,33 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
 
+
+Test the configuration
+----------------------
+
+You don’t have to perform any action. Wazuh detects the vulnerable packages installed in the Ubuntu endpoint automatically. The time it takes to detect vulnerabilities depends on the Syscollector :doc:`interval </user-manual/reference/ossec-conf/wodle-syscollector>` configured in the ``ossec.conf`` file.
+
+#. Install ``vim`` in the Debian endpoint with ``sudo apt install vim -y``.
+
+#. Wait until the syscollector runs a new scan.
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:63]:
```suggestion
```

File: source/proof-of-concept-guide/poc-vulnerability-detection.rst [hunk 8]
```diff
@@ -51,12 +52,33 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
...
+
+#. Delete the package with ``sudo apt purge vim -y``.
```
Comment (javimed) [inline source/proof-of-concept-guide/poc-vulnerability-detection.rst:65]:
```suggestion
#. Delete the package with ``sudo apt purge vim -y`` to fix the vulnerability.
```

**Review by javimed — APPROVED** [review]

**Review by MiguelazoDS — APPROVED** [review]

**Pull request merged**